src/LtiMessageLaunch.php

```diff
@@ -456,7 +451,8 @@ private function validateJwtSignature()
 
         // Validate JWT signature
         try {
-            JWT::decode($this->request['id_token'], $public_key, ['RS256']);
+            $headers = new \stdClass();
+            JWT::decode($this->request['id_token'], $public_key, $headers);
```

Comment:
This was the only change I made to the actual code. Looking at their documentation, it appears we were doing this wrong.

Reply:
Seems like a sensible change looking at https://github.com/firebase/php-jwt.